benchmarks/demo.json

```diff
@@ -0,0 +1 @@
0]}},"ECPR 25":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,90.32,92.31,94.83,95.65,96.43,97.87,100,null,100]}},"ECPR 26":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,78.57,81.82,84.62,86.84,88.94,90.63,92.7,95.12,100]}},"ECPR 27":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,67.05,72.79,76.26,79.31,81.82,85.96,90.4,94.15,100]}},"ECPR 28":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,57.93,68.04,77.55,83.04,86.9,90,93.04,95.62,100]}},"GIQIC 1":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,28.99,32.19,35.76,38.81,41.93,45.75,49.3,54.7,100]}},"GIQIC 10":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,68.37,90,97.92,100,null,null,null,null,100]}},"GIQIC 12":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,82.27,86.06,89.07,91.35,93.28,94.81,96.6,98.45,100]}},"GIQIC 14":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,17.47,21.15,26.77,39.58,53.01,69.23,75.43,82.76,100]}},"GIQIC 15":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,68.18,74.33,77.88,80.71,85.24,89.18,91.35,94.39,100]}},"GIQIC 2":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,91.43,93.73,95.57,96.65,97.47,98.12,98.68,99.28,100]}},"GIQIC 3":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,95.36,96.83,97.5,98.03,98.51,98.86,99.23,99.58,100]}},"GIQIC 4":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,96.46,97.62,98.31,98.77,99.11,99.44,99.69,100,100]}},"GIQIC 5":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,null,null,null,null,null,null,null,0,0]}},"GIQIC 6":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,82.14,87.01,89.66,91.84,93.75,95.1,96.6,98.21,100]}},"GIQIC 8":{"qcdrOrQualifiedRegistry":{"benchmarkYear":"2016","performanceYear":"2018","submissionMethod":"qcdrOrQualifiedRegistry","deciles":[null,null,null,null,null,null,null,null,0,0]}},"GIQIC 9":{"q
```

Comment:
@marimiyachi By flat do you mean just an array of benchmark objects? Not a problem. I just figured this format would make it easier when we have to match the measures with the benchmarks.
```
{
  [qualityId]: {
    [submissionType]: Benchmark
}
```